File: Data_Structure_Topics/Linear_Search.py
# Linear Search

# finding function !
def findTarget(arr, target,len_input):
    for one in range(len_input):
        if arr[one] == target:
            return one
    return -1

# Execute start from here !

len_input = int(input("Enter the number of inputed value : "))
myArr = []
count = 0

while len_input != 0:
    count += 1
    x = int(input(f"SL {count}, Enter value : "))
    myArr.append(x)
    len_input -= 1

target = int(input("Enter the target value : "))

result = findTarget(myArr, target,len(myArr))

if result == -1:
    print("Item Not Found !")
else:
    print(f"Item found at myArr[{result}] = {myArr[result]}")

File: Data_Structure_Topics/Linear_Search.js
// Linear Search

const findTarget = (arr,len,target)=>{
    for(let one=0;one<len;one++){
        if(arr[one]==target){
            return one;
        }
    }
    return -1;
}



const myArr = [2,3,4,1,9,7];
const target = 1;
const len = myArr.length;
const result = findTarget(myArr,len,target);

{
    result == -1 ? console.log("Item Not Found !") : console.log(`Item Found on myArr[${result}]=${myArr[result]}`);
}